### DISCUSSION TITLE
No luck so far

### DISCUSSION THREAD

**Discussion by Dan-imal** [berrystephenw/OctoText#76] [General]
Hi,
I'm not able to send a test message.  I'm running 1.5.3, waiting a bit on1.6.  Hopefully that's not the issue.

I created a new outlook.com email address and confirmed it works.  I've checked and unchecked the box that forces use of 587 and I'm using smtp.mail.outlook.com.  I also tried disabling the webcam snapshots.

But, I keep getting this error: Exception while talking to your mail carrier, check SMTP settings.

I've tried all the basic things I can think of.  I've restarted Octoprint, and rebooted the system.  Not really sure what else I should try but I'm open to suggestions.

Thank you

**Comment by berrystephenw** [comment]
Hi Dan,
I'm not sure what you mean about unchecking the box that uses 587, the checkbox below the SMTP port is for the encryption type, the port is always used. 
If you used the defaults you should be all set - so I would make sure that the SMTP settings look like this:
![image](https://user-images.githubusercontent.com/44931130/116759649-d7030580-a9e0-11eb-950b-07653243863e.png)

Also, if that doesn't work, please send me a logfile and let me know if you are running the rc (release candidate) software or the stable software. If you didn't change anything after installing the software then you are probably running stable.

**Comment by Dan-imal** [comment]
Thank you for the fast reply.  I think what fixed it was putting this in the smtp "smtp.office365.com".

Thank you again.

**Comment by tj51** [comment]
thank you for this doc, may I suggest you replace your some of your read me and faq with this information, I have been working for hours on 2 pi's and 2 minutes with this information , done, thank you again, Tj

**Reply by berrystephenw** [reply]
I will be issuing a new release soon with all of the small changes that have piled up - but if you could tell me what exactly you had a problem with and I'll make it more clear in the Readme. Thanks.
